crates/biome_cli/src/execute/traverse.rs

```diff
@@ -746,6 +750,9 @@ impl<'ctx, 'app> TraversalContext for TraversalOptions<'ctx, 'app> {
             TraversalMode::Lint { .. } => file_features.supports_lint(),
             // Imagine if Biome can't handle its own configuration file...
             TraversalMode::Migrate { .. } => true,
+            // We do need to have a suitable parser, but in principle, every
+            // file is searchable.
```

Comment:
This isn't true. Biome inspects all files, even the ones that can't handle `.png`, `.rs`... any file extension that can't handle. Having `true` here means that it can **search** any file, which isn't the case. Maybe `false` is a better default for now

Reply:
How does that differ from the migrate command, which also uses `true` here? Surely it wouldn’t migrate images either. My thinking was that every file can be passed to the search command, and depending on the language of the pattern to match, the command will determine if that file should be parsed at all.

Reply:
> How does that differ from the migrate command, which also uses true here? 

The `migrate` command doesn't do any traversal. It just reads the `biome.json`/`biome.jsonc` file (and few more), but it doesn't read files that can or can't handle. It always know how to handle the files that reads.

> My thinking was that every file can be passed to the search command, and depending on the language of the pattern to match, the command will determine if that file should be parsed at all.

This is the same assumption for the other features, and then the value changes based on the file's extension.

Reply:
Gotcha 👍